server/etcdserver/api/membership/cluster.go

```diff
@@ -831,3 +837,18 @@ func (c *RaftCluster) updateMembershipMetric(peer types.ID, known bool) {
 	}
 	knownPeers.WithLabelValues(c.localID.String(), peer.String()).Set(v)
 }
+
+// ValidateMaxLearnerConfig verifies the existing learner members in the cluster membership is less than maxLearners.
```

Comment:
I think the doc should say "less than or equal" or "no more than maxLearners" 
e.g. verifies the existing learner members in the cluster membership are not more than maxLearners.